pyresample/_cartopy.py

```diff
@@ -93,7 +93,7 @@ def y_limits(self):
     @property
     def threshold(self):
         x0, x1, y0, y1 = self.bounds
-        return min(x1 - x0, y1 - y0) / 100.
+        return min(np.fabs(x1 - x0), np.fabs(y1 - y0)) / 100.
```

Comment:
Should this maybe use `math` instead of numpy since these are scalars? Or is there another builtin fabs? Does `abs` work the same?